src/integration/pages/storyPage/crossPlatformTests.js

```diff
@@ -12,8 +12,62 @@ export default () => {
     expect(imageEl).toBeInTheDocument();
     expect(imageCaptionEl).toBeInTheDocument();
     expect(imageCaptionEl.textContent).toBeTruthy();
-    expect(imageCaptionEl.textContent).toMatchInlineSnapshot(
-      `"Pie de foto, Llegó el día de la salida de Reino Unido de la UE."`,
+    expect(imageCaptionEl.textContent).toMatchSnapshot();
+  });
+
+  /**
+   * Social Embeds.
+   */
+  it('I can see the skip social embed link', () => {
+    const skipLinkEl = document.querySelector(
+      'a[href="#skip-youtube-content-1"]',
     );
+
+    expect(skipLinkEl).toBeInTheDocument();
+    expect(skipLinkEl.textContent).toBeTruthy();
+    expect(skipLinkEl.textContent).toMatchSnapshot();
+  });
+
+  it('I can see the skip social embed link destination', () => {
+    const skipLinkDestinationEl = document.getElementById(
+      'skip-youtube-content-1',
+    );
+
+    expect(skipLinkDestinationEl).toBeInTheDocument();
+    expect(skipLinkDestinationEl.textContent).toBeTruthy();
+    expect(skipLinkDestinationEl.textContent).toMatchSnapshot();
   });
+
+  // Social Embeds - Rich.
+  const hasRichYouTubeEmbed = [
+    document.querySelector('iframe[src*="https://www.youtube.com/embed/"]'),
+    document.querySelector('amp-youtube'),
+  ].find(el => !!el);
+
+  if (hasRichYouTubeEmbed) {
+    it('I can see the caption accompanying a rich YouTube embed', () => {
+      const figCaptionEl = document.querySelector(
+        'a[href="#skip-youtube-content-1"] + figure > figcaption',
+      );
+
+      expect(figCaptionEl).toBeInTheDocument();
+      expect(figCaptionEl.textContent).toBeTruthy();
+      expect(figCaptionEl.textContent).toMatchSnapshot();
+    });
+  }
+
+  // Social Embeds - Fallback.
+  const hasFallback = !!document.getElementById('skip-facebook-content-1');
+
+  if (hasFallback) {
+    it('I can see a link to the original content in a social embed fallback', () => {
+      const externalLinkEl = document.querySelector(
+        'a[href="https://www.facebook.com/228735667216/posts/10157712846517217/?sfnsn=scwspmo&extid=ixkwkArH1fy0g2GP"]',
```

Comment:
Is this always going to be the same for every STY with social embed fallback?

Reply:
It's not - good spot. This test is more to assert a link exists rather than it being well-formed, so I've loosened off the selector to allow partial matching.